F2_Operações de Crédito - Financiamentos (Financings)/get-contracts-{contractId}/get-financings-contracts-contractId-1.1.json

```diff
@@ -1,60 +1,58 @@
 {
   "data": {
-    "contractNumber": "92525769252",
-    "ipocCode": "42061493000110792118751740044492525769252",
+    "contractNumber": "34589374320",
+    "ipocCode": "00000000040118751740044439873689375023953",
     "productName": "Crédito Pessoal Consignado",
     "productType": "FINANCIAMENTOS",
     "productSubType": "AQUISICAO_BENS_VEICULOS_AUTOMOTORES",
-    "contractDate": "2021-01-15",
-    "disbursementDate": "2021-01-29",
-    "settlementDate": "2021-01-15",
-    "contractAmount": 105000.40,
+    "contractDate": "2021-06-09",
+    "settlementDate": "NA",
+    "contractAmount": 50000.0000,
     "currency": "BRL",
-    "dueDate": "2028-01-15",
+    "dueDate": "2026-07-09",
     "instalmentPeriodicity": "MENSAL",
-    "instalmentPeriodicityAdditionalInfo": "",
-    "firstInstalmentDueDate": "2021-02-15",
-    "CET": 0.29,
-    "amortizationScheduled": "SAC",
+    "instalmentPeriodicityAdditionalInfo": "NA",
+    "firstInstalmentDueDate": "2021-07-09",
+    "CET": 0.5858,
+    "amortizationScheduled": "SEM_SISTEMA_AMORTIZACAO",
     "amortizationScheduledAdditionalInfo": "NA",
     "interestRates": [
       {
         "taxType": "NOMINAL",
         "interestRateType": "COMPOSTO",
-        "taxPeriodicity": "AA",
+        "taxPeriodicity": "AM",
         "calculation": "21/252",
-        "referentialRateIndexerType": "PRE_FIXADO",
+        "referentialRateIndexerType": "POS_FIXADO",
         "referentialRateIndexerSubType": "TJLP",
-        "referentialRateIndexerAdditionalInfo": "",
-        "preFixedRate": 0.6,
-        "postFixedRate": 0.55,
-        "additionalInfo": ""
+        "preFixedRate": 0.0000,
+        "postFixedRate": 0.5858,
+        "additionalInfo": "NA"
       }
     ],
     "contractedFees": [
       {
-        "feeName": "Excesso em Conta",
-        "feeCode": "EXCESSO_CONTA",
-        "feeChargeType": "UNICA",
-        "feeCharge": "MINIMO",
-        "feeAmount": 105000.40,
-        "feeRate": 0.2
+        "feeName": "Taxa de administração",
+        "feeCode": "ADMINISTRACAO",
+        "feeChargeType": "POR_PARCELA",
+        "feeCharge": "Percentual",
```

Comment:
Percentual precisa preencher feeRate e deixar feeAmount 0.0000, ajustar os valores ou alterar o feeCharge

Reply:
Se for percentual, preencher feeRate e somar o valor dele no CET